src/transi/transi_module.F90

```diff
@@ -501,13 +541,40 @@ function trans_set_radius(radius) bind(C,name="trans_set_radius")
   trans_set_radius = TRANS_SUCCESS
 end function
 
+function trans_set_leq_regions(ldeq_regions) bind(C,name="trans_set_leq_regions")
+  integer(c_int) :: trans_set_leq_regions
+  BOOLEAN, value, intent(in) :: ldeq_regions
+  if( ldeq_regions == 0) then
+    LEQ_REGIONS = .false.
+  else
+    LEQ_REGIONS = .true.
+  endif
+  trans_set_leq_regions = TRANS_SUCCESS
+end function
+
 function trans_set_nprtrv(kprtrv) bind(C,name="trans_set_nprtrv")
   integer(c_int) :: trans_set_nprtrv
   integer(c_int), value, intent(in) :: kprtrv
   NPRTRV = kprtrv
   trans_set_nprtrv = TRANS_SUCCESS
 end function
 
+function trans_set_nprgpew(kprgpew) bind(C,name="trans_set_nprgpew")
+  integer(c_int) :: trans_set_nprgpew
+  integer(c_int), value, intent(in) :: kprgpew
+  ! only possible before trans_init
+  if ( .not. is_init ) then
+    NPRGPEW = kprgpew
+  else
+    if(NPRGPEW /= KPRGPEW) then
+      write(error_unit,'(A,I0,A)')"trans_set_nprgpew: Must be called before trans_init, and may not be modified later (nprgpew=",NPRGPEW,")"
```

Comment:
According to the CI, this line is too long for GNU. I think the limit is 132 characters. We could add `-ffree-line-length-none` but I think it would be easier just to chop this line up. Here's a suggestion:

```suggestion
      write(error_unit,'(A)')"trans_set_nprgpew: Must be called before trans_init, and may not be modified later"
      write(error_unit,'(A,I0,A)') "(nprgpew=",NPRGPEW,")"
```